comps/dataprep/opensearch/langchain/config.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+# Embedding model
+EMBED_MODEL = os.getenv("EMBED_MODEL", "BAAI/bge-base-en-v1.5")
+
+# OpenSearch Connection Information
+OPENSEARCH_HOST = os.getenv("OPENSEARCH_HOST", "localhost")
+OPENSEARCH_PORT = int(os.getenv("OPENSEARCH_PORT", 9200))
+OPENSEARCH_INITIAL_ADMIN_PASSWORD = os.getenv("OPENSEARCH_INITIAL_ADMIN_PASSWORD", "StRoNgOpEa0)")
```

Comment:
Please check if this is your personal password and hide it.

Reply:
I did need to leave this password in the test scripts since they require some default password to be used when spinning up the opensearch vectorstore container, but I removed their reference from all other files such as this one. Great comment, thank you for catching this!